tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "include": [
         "typings/index.d.ts",
         "src/**/*.ts",
-        "src/**/*.tsx"
+        "src/**/*.tsx",
+        "dangerfile.ts"
     ],
     "exclude": [
         "dangerfile.ts",
```

Comment:
I discovered this trick when building the yeoman generator. The Dangerfile can be included by the TSServer (dev-time) and ignored by TSC (build-time).